Dockerfile.alltools

```diff
@@ -8,6 +8,12 @@ FROM ethereum/client-go:v1.8.27 as geth
 
 FROM alpine:3.9
 RUN apk --no-cache add ca-certificates
+RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
+RUN apk --update add perf
+RUN wget https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+RUN chmod +x /jq-linux64
```

Comment:
I am not sure why we need `jq` and `websocat` within the image. What is the use-case here? I use those tools from my local machine, why do we need them in the `swarm` binary?